site/src/pages/docs/server-annotated-service.mdx

```diff
@@ -1428,3 +1428,71 @@ for more information about RPC and HTTP service names, and how to use it.
 </Tip>
 
 [Publisher]: https://www.reactive-streams.org/reactive-streams-1.0.2-javadoc/org/reactivestreams/Publisher.html
+
+## Kotlin Coroutines Support
+
+You can implement annotated services with Kotlin Coroutines.
+
+```kotlin
+class MyAnnotatedService {
+    @Get("/users/{name}")
+    suspend fun getUserToken(@Param("name") name: String): HttpResponse {
+        val user = myReactiveRepository.findByName(name).await()
+        return HttpResponse.of(user.token)
```

Comment:
It would be more natural to return `String` for annotated services if don't need to control HTTP status or other headers.